mandatory/2-function-creation.js

```diff
@@ -5,7 +5,13 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  let arrOfStrings = ["   CodeYourFuture/Homework/Mandatory   ","Douglas/  ", "  /douGlas"," dougLas "];
```

Comment:
You forgot to remove your test array and make your function process passed parameters

Reply:
Thank you Simba.
I created the pull request after completing the exercises.
I'm still working on the mandatory folder.